json/src/main/kotlin/com/trib3/json/ObjectMapperProvider.kt

```diff
@@ -45,8 +53,20 @@ class ObjectMapperProvider @Inject constructor(
     }
 
     override fun get(): ObjectMapper {
-        val mapper = Jackson.newObjectMapper()
-        mapper.disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES)
+        // Start with a minimal mapper, then register everything from Jackson.newObjectMapper()
+        // EXCEPT blackbird due to a race condition issue:
+        // https://github.com/FasterXML/jackson-modules-base/issues/169
+        val mapper = Jackson.newMinimalObjectMapper()
+        mapper.registerModule(GuavaExtrasModule())
+        mapper.registerModule(CaffeineModule())
+        mapper.registerModule(JodaModule())
+        mapper.registerModule(FuzzyEnumModule())
+        mapper.registerModule(ParameterNamesModule())
+        mapper.registerModule(Jdk8Module())
+        mapper.registerModule(JavaTimeModule())
+        mapper.propertyNamingStrategy = AnnotationSensitivePropertyNamingStrategy()
+
+        // additional leakycauldron configuration:
```

Comment:
should be able to avoid this change with the 2.13.3 jackson release, holding off on this dropwizard upgrade until then